src/main/java/com/gluonhq/substrate/target/AbstractTargetConfiguration.java

```diff
@@ -359,6 +360,14 @@ private String getJniPlatformArg() {
 
     private String getJniPlatform() {
         Triplet target = projectConfiguration.getTargetTriplet();
+        boolean graalVM221 = false;
+        try {
+            Version graalVersion = projectConfiguration.getGraalVersion();
+            if ((graalVersion.getMajor() > 21) && (graalVersion.getMinor() >0)) graalVM221 = true;
+        } catch (IOException ex) {
+            ex.printStackTrace();
```

Comment:
I don't think we need an stacktrace here?

Reply:
why not? This should never happen, so if it happens, we at least need a detailed reason (or we should call System.exit())